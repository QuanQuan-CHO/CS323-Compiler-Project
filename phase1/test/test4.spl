#include <stdio.h>

/*
Please disable ASLR by sudo sysctl -w kernel.randomize_va_space=0
*/

void win() {  // at 0x08048456
    puts("Excellent, now let's go hack the world");
}
void vuln() {
    char buf[16];
    scanf("%s", buf);
}
int main() {
    puts("Welcome back to 2023 CS315, let's have some fun!");
    vuln();
    puts("Have a good day, Bye~");
    return 0;
}