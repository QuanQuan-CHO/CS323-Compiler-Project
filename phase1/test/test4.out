INCLUDE (1)
  stdio.h
Program (1)
